src/rdma.c

```diff
@@ -77,8 +77,11 @@ typedef enum ValkeyRdmaOpcode {
 #define VALKEY_RDMA_INVALID_OPCODE 0xffff
 #define VALKEY_RDMA_KEEPALIVE_MS 3000
 
+#define RDMA_CONN_FLAG_POSTPONE_UPDATE_STATE (1 << 0)
+
 typedef struct rdma_connection {
     connection c;
+    int flags;
```

Comment:
I guest
```
struct rdma_cm_id *cm_id;
int flags;
int last_errno;
```

would reduce 8 bytes.

Otherwise LGTM. I also test it locally, it works fine.

Hi @zuiderkwast , would you please take a look at this change?

Reply:
:) Thanks a lot for the detailed feedback!